src/main/java/com/spring/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,34 @@
+package com.spring.onlinebookstore.model;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "books")
+@Data
```

Comment:
```suggestion
@Getter
@Setter
```

don't use `@Data` for entities